util/linux/attract-mode.desktop

```diff
@@ -1,9 +1,9 @@
 [Desktop Entry]
-Version=1.0
+Version=2.3.0
```

Comment:
This field isn't for the version of the program.

From the [Desktop Entry Specification](https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#recognized-keys):
> Version of the Desktop Entry Specification that the desktop entry conforms with. Entries that confirm with this version of the specification should use 1.1. Note that the version field is not required to be present.